examples/celery/docker-compose.yml

```diff
@@ -37,6 +37,11 @@ services:
     volumes:
       - ./superset:/etc/superset
     command: "celery worker --app=superset.tasks.celery_app:app"
+    healthcheck:
+      test: curl -f http://superset:8088/health || exit 1
+      interval: 1m
+      timeout: 10s
+      retries: 3    
```

Comment:
shouldn't the healthcheck be monitoring the state of the worker? Is there a way that can be done?

Reply:
worker  is celery worker, so

something like: celery -A [app_name] status should be ok. (I did not verify)

https://stackoverflow.com/questions/8506914/detect-whether-celery-is-available-running